src/itemdef.cpp

```diff
@@ -214,12 +221,15 @@ void ItemDefinition::deSerialize(std::istream &is, u16 protocol_version)
 	inventory_overlay = deSerializeString16(is);
 	wield_overlay = deSerializeString16(is);
 
-	// If you add anything here, insert it primarily inside the try-catch
+	// If you add anything here, insert it inside the try-catch
 	// block to not need to increase the version.
 	try {
 		short_description = deSerializeString16(is);
 
 		place_param2 = readU8(is); // 0 if missing
+
+		sound_use.deSerialize(is, CONTENTFEATURES_VERSION);
+		sound_use_air.deSerialize(is, CONTENTFEATURES_VERSION);
```

Comment:
```suggestion
		sound_use.deSerialize(is, protocol_version);
		sound_use_air.deSerialize(is, protocol_version);
```
Rebase issue caused by versioning changes in the meantime (although it does functionally not matter as of now).

Reply:
hmm I'm pretty sure I fixed this conflict